api/python/quilt3/telemetry.py

```diff
@@ -15,19 +15,26 @@
 TELEMETRY_URL = "https://telemetry.quiltdata.cloud/Prod/metrics"
 TELEMETRY_USER_AGENT = "QuiltCli"
 TELEMETRY_CLIENT_TYPE = "quilt3-python-client"
-TELEMETRY_SCHEMA_VERSION = "pyclient-usage-metrics-v1"
+TELEMETRY_SCHEMA_VERSION = "pyclient-usage-metrics-v2"
```

Comment:
We should use semver or schemaversion for this, but not a blocking concern
https://snowplowanalytics.com/blog/2014/05/13/introducing-schemaver-for-semantic-versioning-of-schemas/#schemaver

I think this counts as an ADDITION so `...-v1.0.1`.

The original string is silly but that's a separate problem.